From 34123084a3bf65c9bd4606d46f69f47328e469bc Mon Sep 17 00:00:00 2001
From: Maxime Roussin-Belanger <maxime.roussinbelanger@gmail.com>
Date: Tue, 11 Jun 2024 14:00:25 -0400
Subject: [PATCH] common: namespace: add missing include QList

Otherwise Q_DECLARE_METATYPE can't see the QList type and complains
about incomplete type

Upstream-Status: Submitted [https://github.com/maliit/framework/pull/122]
---
 common/maliit/namespace.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/maliit/namespace.h b/common/maliit/namespace.h
index 2e47b17..a4f2750 100644
--- a/common/maliit/namespace.h
+++ b/common/maliit/namespace.h
@@ -16,6 +16,7 @@
 
 #include <QMetaType>
 #include <QSharedPointer>
+#include <QList>
 
 //! \ingroup common
 namespace Maliit {
